cmd/gateway-main.go

```diff
@@ -193,7 +191,7 @@ func gatewayMain(ctx *cli.Context) {
 			cert, key = getPublicCertFile(), getPrivateKeyFile()
 		}
 		if aerr := apiServer.ListenAndServe(cert, key); aerr != nil {
-			console.Fatalf("Failed to start minio server. Error: %s\n", aerr)
+			fatalIf(err, "Failed to start minio server")
 		}
```

Comment:
It should be `aerr` instead of `err` in fatalIf, and this block can be rewritten in fewer lines as:

```go
aerr := apiServer.ListenAndServe(cert, key)
fatalIf(aerr, "Failed to start minio server")
```